examples/wafv2-custom-response/main.tf

```diff
@@ -0,0 +1,77 @@
+provider "aws" {
+  region = "eu-west-1"
+}
+
+module "waf" {
+  source = "../.."
+
+  name_prefix          = var.name_prefix
+  allow_default_action = true
+
+  create_alb_association = false
+
+  visibility_config = {
+    cloudwatch_metrics_enabled = false
+    metric_name                = "${var.name_prefix}-waf-setup-waf-main-metrics"
+    sampled_requests_enabled   = false
+  }
+
+  custom_response_bodies = [
+    {
+      key          = "custom_response_body_1",
+      content      = "You are not authorized to access this resource.",
+      content_type = "TEXT_PLAIN"
+    },
+    {
+      key          = "custom_response_body_2",
+      content      = "You there, are not authorized to access this resource.",
+      content_type = "TEXT_PLAIN"
+    }
+  ]
+
+  rules = [
```

Comment:
`visibility_config` will be required in both of the rules - see here: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/wafv2_web_acl#visibility_config

Reply:
Thanks for the catch here as well... nuked the `visibility_config` from the copy of my personal block because had local variables I didn't want to bother with here. But just added the block ✔️ .